modules/oc-mirror-setting-incremental-mirroring.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="setting-incremental-mirroring_{context}"]
+= Setting up for incremental mirroring
+
+You mirror images to install, upgrade, and maintain {product-title} in a disconnected environment. Because the volume of the image set needed to install or upgrade {product-title} can be substantial, oc-mirror plugin v1 stores the metadata of the previously mirrored images in its storage backend thus reducing the volume of the image sets to the recent images only.
+
+You can specify the following storage backends:
+
+* A local directory 
++
+[NOTE]
+====
+Using a local directory is not recommended for enterprise scenarios because it does not scale effectively. This method is suitable only for small-scale environments or testing purposes.
+====
++
+.Example `ImageSetConfiguration` file
+[source,terminal]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+storageConfig:                                                    
+  local:
+    path: <oc_mirror_workspace> <1>
+----
+<1> Specify your target workspace path.
+* A registry that supports Docker v2-2
```

Comment:
Is this the right place for this statement ? we are only talking about the localpath right, so why `registry` then ?